tests/integration/modules/beacons.py

```diff
@@ -0,0 +1,63 @@
+# coding: utf-8
+
+# Python Libs
+from __future__ import absolute_import
+
+# Salt Libs
+from salt.modules import beacons
+import integration
+from salttesting import skipIf
+from salttesting.helpers import destructiveTest, ensure_in_syspath
+
+ensure_in_syspath('../../')
+
+beacons.__opts__ = {}
+
+
+class BeaconsTest(integration.ModuleCase):
+    '''
+    Tests the beacons execution module
+    '''
+    @destructiveTest
+    def test_add(self):
+        ret = self.run_function('beacons.add', ['ps', {'apache2': 'stopped'}])
+        self.assertEqual(ret, {'comment': 'Added beacon: ps.', 'result': True})
```

Comment:
This shows a correct return, but two comments.

First, this is kind of brittle because it checks the comment. If somebody makes a tiny grammatical change, this will cause the test to fail. Might be better just to test for the result.

Second, ideally it would be better to actually check to make sure the beacon was really added and running. Maybe if we had a test beacon that could write a timestamp to a tmp file or something (this would be a good idea regardless) and then we could use that to see if our beacon operations were actually doing what we expect.